src/lib/resource/sections/applicant-claim-type.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
                 },
                 'funeral-costs-only-info': {
```

Comment:
should this be applicant-claim-type-info?

Reply:
happy to leave as is to reduce cica-web changes
https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/403f7c98030c427c8a44389c13dfafd2beab6516/questionnaire/questionnaireUISchema.js#LL1802C13-L1802C13

Reply:
Changed